### PR TITLE
Adding parameter dataloading_s to console logs and wandb for tracking…

### PR DESCRIPTION
**Pull request by amandip7**
… the time it takes to load each batch of data while training

## What this does
Adds parameter dataloading_s to wandb logs and console logs. This implements the changes required for [this item](https://github.com/orgs/huggingface/projects/46/views/1?pane=issue&itemId=64693357)

## How it was tested
train.py run along with INFO logging was used to verify the new parameter's presence in wandb and console logs

## How to checkout & try? (for the reviewer)
command used to run : python lerobot/scripts/train.py wandb.enable=true training.offline_steps=10000 training.log_freq=10 training.eval_freq=25 training.save_freq=250

- - -
This change is [Reviewable](https://reviewable.io/reviews/huggingface/lerobot/243)

### CODE REVIEW

File: lerobot/scripts/train.py
Comment (alexander-soare) [inline lerobot/scripts/train.py:174]: Can we please be consistent with `dloading_s` vs `dataloading_s` (preferring the second one as it is clearer).

**Review by alexander-soare — APPROVED** [review]
Thanks so much for this! Just left a `nit` on naming. Also waiting on @Cadene to review.

File: lerobot/scripts/train.py
Comment (amandip7) [reply lerobot/scripts/train.py]: This update is done.

Comment (alexander-soare) [reply lerobot/scripts/train.py]: Looking good! Thank you

Comment (Cadene) [inline lerobot/scripts/train.py]:
```suggestion
        f"data_s:{dataloading_s:.3f}",  # if not ~0, you are bottlenecked by cpu or io
```

**Review by Cadene — CHANGES_REQUESTED** [review]
Left a suggestion.

File: lerobot/scripts/train.py
Comment (alexander-soare) [reply lerobot/scripts/train.py]: Agreed on this. Good shout @Cadene

File: lerobot/scripts/train.py
Comment (Cadene) [inline lerobot/scripts/train.py:174]:
```suggestion
        f"data_s:{dataloading_s:.3f}",  # if not ~0, you are bottlenecked by cpu or io
```

**Review by Cadene — APPROVED** [review]

File: lerobot/scripts/train.py
Comment (Cadene) [reply lerobot/scripts/train.py:174]: @amandip7 could you apply this suggestion? and let's merge :)

Comment (amandip7) [reply lerobot/scripts/train.py]: Done @Cadene Thanks!

**Comment by amandip7** [comment]
@alexander-soare @Cadene Could you please approve the pending workflows? Thank you!

**Pull request merged**
